functional/jtreg-buffer/playlist.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+     xsi:noNamespaceSchemaLocation="../../TKG/resources/playlist.xsd">
+     <test>
+          <testCaseName>jtreg-buffer</testCaseName>
+          <disables>
+               <disable>
+                    <comment>readlink -f is known to misbehave on selected platforms</comment>
+                    <platform>^.*(s390x_linux|aarch64_mac).*$</platform>
+               </disable>
+          </disables>
+          <command>
+               JTREG_HOME="$(TEST_RESROOT)$(D)jtreg" \
+               $(TEST_ROOT)$(D)functional$(D)jtreg-buffer$(D)jtreg-buffer$(D)run.sh "$(TEST_JDK_HOME)" ; $(TEST_STATUS)
+          </command>
+          <levels>
+               <level>dev</level>
+          </levels>
+          <groups>
+               <group>functional</group>
+          </groups>
+     </test>
+     <test>
```

Comment:
Catch me up on the logic of this, I assumed that we have the 2 targets jtreg-buffer and jtreg-buffer_jtreg to allow vendors to run all tests on all platforms without the run.sh.  The main issue with the run.sh was not because readlink -f misbehaves, but that run.sh may at any point start including other behaviour that is unwanted.

Reply:
In this case it would be https://github.com/rh-openjdk/jtreg-buffer/blob/main/run.sh#L121  which would otherwise be replciated everywhere.  run.sh is from rh-openjdk organization, so it should really not do anything bad. 

(and the https://github.com/rh-openjdk/jtreg-buffer/blob/main/run.sh#L111, which is disabled in aqavit, but hsoudl be allowed to be enabled)